client/src/components/Property/WalkScore.js

```diff
@@ -11,14 +9,11 @@ function WalkScore({ zpid }) {
   const walkAndTransitScore = useSelector(
     state => state.property.walkAndTransitScore
   );
+
   useEffect(() => {
     dispatch(getWalkAndTransitScoreAsync({ zpid, token }));
   }, [zpid]);
 
-  useEffect(() => {
-    //
-  }, [walkAndTransitScore]);
```

Comment:
@pukkuKim  we need this line for page rendering

Reply:
Restored!